src/app/[language]/calculator/[visa]/page.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+import { useFormConfig } from '@lib/hooks'
+import { RedirectType, redirect } from 'next/navigation'
+
+interface Props {
+  params: {
+    visa: string,
+    language: string,
+  }
+}
+
+export default function Page({ params }: Props) {
+  const formConfig = useFormConfig(params.visa)
+  const language = params?.language
```

Comment:
We'll want a fallback language or we might risk having `undefined` in the url.

Reply:
true, although for this particular file we won't need to pass language since middleware will automatically include the language in the params